data/src/main/java/org/apache/iceberg/data/IcebergGenerics.java

```diff
@@ -66,6 +67,11 @@ public ScanBuilder select(String... selectedColumns) {
       return this;
     }
 
+    public ScanBuilder select(Collection<String> columns) {
+      this.tableScan = tableScan.select(ImmutableList.copyOf(columns));
```

Comment:
Just wondering do we really need a copy of the collection again? Can't we pass the collection directly as the below interface supports it?
https://github.com/apache/iceberg/blob/master/api/src/main/java/org/apache/iceberg/Scan.java#L87

Reply:
In my opinion, there is no need to copy, but I am not sure why the previous author did this? Is there any special purpose?

Reply:
they may want to convert arrays into a collection and use the collection interface. But that is also not needed now as the underlying interface has a conversion logic. 

https://github.com/apache/iceberg/blob/634215fcd411dd8a6d5c0b6d7d8dbf4eb66494d8/api/src/main/java/org/apache/iceberg/Scan.java#L96-L98

Reply:
get it, i'll change there